### PR TITLE
Update user guide

### LINKED ISSUE #236: [PE-D] Bookmarked incomes or expenses are still there even when the income or expense entry is deleted
After deleting an item from the income or expense list, it is still being listed in the bookmarks tab. There is a need to separately delete the book mark via a delete-bookmark command, which is an unneccesary extra step. In the example below, the Teaching Assistant entry was in both bookmarked income and income. After deleting the expense from the income list, the bookmarked expense is still in the bookmarked list.

-------------
Labels: `severity.Low` `type.FeatureFlaw`
original: tengjianling/ped#1

### LINKED ISSUE #244: [PE-D] Typo in the user guide
In section 4.5.1, the expected outcome should have an amount of `$65.00` instead of `$60.00`.

Literally unusable 0/10 user guide.

-------------
Labels: `severity.VeryLow` `type.DocumentationBug`
original: Wong-ZZ/ped#3

### LINKED ISSUE #247: [PE-D] Maximum number of digits for amount is unclear
-- says can accept up to 8 digits but did not specify whether include decimal places or not (with decimal places it can be 10 digits)

E.g.
`add-expense t/Starbucks Coffee a/12345678.12`

-------------
Labels: `severity.Low` `type.DocumentationBug`
original: aizatazhar/ped#1

### LINKED ISSUE #248: [PE-D] Quick start instruction unclear
Following the quick start instructions step-by-step, I expected the delete command in step 5 to delete the list currently shown (overview) but delete is only allowed on expenses and incomes tabs. Maybe can include this information in the quick start step (since the quick start says it deletes from the current list which is overview list)

-------------
Labels: `severity.VeryLow` `type.DocumentationBug`
original: aizatazhar/ped#2

### PR DESCRIPTION
**Pull request by yongping827**
Related to #131.
Resolves #236.
Resolves #244.
Resolves #247.
Resolves #248.

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -167,7 +167,8 @@ This section aims to provide you with in-depth details on Fine\$\$e's unique fea
 
 The formats of the parameters used in the rest of the document are as follows:
 * `TITLE` and `CATEGORY` should consist of <abbr title="Alphanumeric characters, space, and the special characters !&quot;#$%&'()*+,-./:;&lt;=&gt;?@[\]^_`{\|}~">printable ASCII characters</abbr>, and cannot begin with a space.
-* `AMOUNT`, `AMOUNT_FROM` and `AMOUNT_TO` should be non-negative numbers up to 8 digits with 0 or 2 decimal places, with an optional `$` in front.
+* `AMOUNT`, `AMOUNT_FROM` and `AMOUNT_TO` should be non-negative numbers with up to 8 digits before yje decimal point and with 0 or 2 decimal places
```
Comment (wltan) [inline docs/UserGuide.md:170]:
```suggestion
* `AMOUNT`, `AMOUNT_FROM` and `AMOUNT_TO` should each be a non-negative number with up to 8 digits before the decimal point and with 0 or 2 decimal places after.
```

**Review by wltan — CHANGES_REQUESTED** [review]
Small problems with the fix for #247, but the rest look fine. Note that #244 has already been resolved.

File: docs/UserGuide.md (hunk 1 above)
Comment (wltan) [reply docs/UserGuide.md]: The changes here were reverted during the force push. #247 is now unresolved.

**Review by zhaojj2209 — DISMISSED** [review]
LGTM!

File: docs/UserGuide.md [hunk 2]
```diff
@@ -173,7 +173,8 @@ The formats of the parameters used in the rest of the document are as follows:
 0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijkl
 mnopqrstuvwxyz !"#$%&'()*+,-./:;<=>?@[\]^_`{|}~
   ```
-* `AMOUNT`, `AMOUNT_FROM` and `AMOUNT_TO` should be non-negative numbers up to 8 digits with 0 or 2 decimal places, with an optional `$` in front.
+* `AMOUNT`, `AMOUNT_FROM` and `AMOUNT_TO` should be non-negative numbers with up to 8 digits before the decimal point and with 0 or 2 decimal places.
+  An optional `$` may be included in front as well.
 * `DATE`, `DATE_FROM` and `DATE_TO` should be a valid calendar date in `dd/mm/yyyy` format, representing day, month, and year respectively.
```
Comment (wltan) [inline docs/UserGuide.md:178]: Inconsistent plurality between amounts and dates.
```suggestion
* `AMOUNT`, `AMOUNT_FROM` and `AMOUNT_TO` should be a non-negative number with up to 8 digits before the decimal point and with 0 or 2 decimal places.
  An optional `$` may be included in front as well.
* `DATE`, `DATE_FROM` and `DATE_TO` should be a valid calendar date in `dd/mm/yyyy` format, representing day, month, and year respectively.
```
\- OR -
```suggestion
* `AMOUNT`, `AMOUNT_FROM` and `AMOUNT_TO` should be non-negative numbers with up to 8 digits before the decimal point and with 0 or 2 decimal places.
  An optional `$` may be included in front as well.
* `DATE`, `DATE_FROM` and `DATE_TO` should be valid calendar dates in `dd/mm/yyyy` format, representing day, month, and year respectively.
```
However, the latter may suggest that multiple numbers or dates can be assigned to each amount/date.

**Review by wltan — CHANGES_REQUESTED** [review]

File: docs/UserGuide.md (hunk 2 above)
Comment (wltan) [reply docs/UserGuide.md:178]: Another alternative using `each`:
```suggestion
* `AMOUNT`, `AMOUNT_FROM` and `AMOUNT_TO` should each be a non-negative number with up to 8 digits before the decimal point and with 0 or 2 decimal places.
  An optional `$` may be included in front as well.
* `DATE`, `DATE_FROM` and `DATE_TO` should each be a valid calendar date in `dd/mm/yyyy` format, representing day, month, and year respectively.
```

**Review by wltan — APPROVED** [review]
LGTM!

**Review by zhaojj2209 — APPROVED** [review]
LGTM!

**Review by ianyong — APPROVED** [review]
LGTM!

**Pull request merged**